pydrawise/client.py

```diff
@@ -326,30 +327,15 @@ async def get_water_flow_summary(
             return SensorFlowSummary(total_water_volume=LocalizedValueType(0.0, "gal"))
         return deserialize(SensorFlowSummary, sensors[0]["flowSummary"])
 
-    async def get_watering_report(
-        self, controller: Controller, start: datetime, end: datetime
+    @staticmethod
```

Comment:
Just make this a top-level function in the module. No need for it to be a staticmethod.

Reply:
Fixed.